lib/dependencies/WebAssemblyImportDependency.js

```diff
@@ -70,6 +74,31 @@ class WebAssemblyImportDependency extends ModuleDependency {
 	get type() {
 		return "wasm import";
 	}
+
+	serialize(context) {
+		const { write } = context;
+
+		write(this.name);
+		write(this.description);
```

Comment:
This might not work, because there is no serializer for `ModuleImportDescription`, but I'll take it for now. This will probably fail when we test with the wasm cases.